vars/githubscm.groovy

```diff
@@ -74,6 +74,24 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def createBranch(String branchName){
+    try {
+        sh "git checkout -b ${branchName}"
+    } catch (Exception e) {
+        println "[ERROR] Can't create branch ${branchName} on repo."
+        throw e;
+    }
+ println "[INFO] Created branch '${branchName}' on repo."
+}
+
+def commitChanges(String userName, String userEmail, String commitMessage){
```

Comment:
I would replace this method by

```
def commitChanges(String userName, String userEmail, String commitMessage, String filesToAdd = '--all'){
    sh "git config user.name '${userName}'"
    sh "git config user.email '${userEmail}' "
    sh "git add ${filesToAdd}"
    sh "git commit -m '${commitMessage}' "
}
```
WDYT?

Reply:
Yes, makes more sense.

Reply:
Updated